lib/item_repository.rb

```diff
@@ -64,7 +71,7 @@ def create(attributes)
   end
 
   def update(id, attributes)
-    find_by_id(id).update(attributes)
+    find_by_id(id)&.update(attributes)
```

Comment:
nice!

Reply:
What's the ampersand doing there?

Reply:
@aschwartz1 it is used so you don't have to use
```ruby
if !find_by_id(id).nil?
``` 
or 
```ruby
unless find_by_id(id).nil?
```

Reply:
RT^
@aschwartz1 You read it as find by id and if there is an id then update.

Reply:
Oh, I see. Neat!